concent_api/core/validation.py

```diff
@@ -468,28 +467,17 @@ def substitute_new_report_computed_task_if_needed(
     return new_report_computed_task
 
 
-def validate_list_of_transaction_timestamp(
-    list_of_transactions: List[BatchTransferEvent],
-    acceptances: List[SubtaskResultsAccepted],
-) -> None:
+def validate_list_of_transaction_timestamp(acceptances: List[SubtaskResultsAccepted]) -> None:
     """
     Validate if cut_off_time < youngest_payment_ts(T2) + PDT, unless requestor already made some payment.
     We compare only youngest payment_ts, because we want to use the highest value of timestamp. Other payment_ts might
     give false positives.
     """
 
     cut_off_time = get_current_utc_timestamp()
-
     youngest_payment_ts = max(subtask_results_accepted.payment_ts for subtask_results_accepted in acceptances)
 
-    if (
-        (
-            any(youngest_payment_ts > closure_time for closure_time in list_of_transactions) or
-            len(list_of_transactions) == 0
```

Comment:
This condition ensured that a timestamp error was returned if there were any payments before the first acceptance? Or am I missing something?

This seems wrong and I don't remember it at the time we were doing #983, so why is it here?

Reply:
Good question, my only guess is that something went wrong during rebase in other pull requests, as I remember that there were some other changes in this part of the code.